index.js

```diff
@@ -100,14 +144,16 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+function triple 
```

Comment:
This one would look like:
```javascript
function triple(num) {
     return num * 3
}
```

The myMap function takes two arguments. The array and a function. It iterates over each item in the array and calls the function on each of those items. Now that you have this solution, examine one line at a time what happens when we call `myMap([2,4,6,8], triple)`